### PR TITLE
Standardize spaces

### PR DESCRIPTION
**Pull request by wkolowski**
This is the same stuff as in https://github.com/runtimeverification/casper-cbc-proofs/pull/510, except this time I also added a script to the `scripts/` directory which can semi-automatically fix the kinds of lexical issues that are handled in this PR.

### CODE REVIEW

File: theories/VLSM/Lib/StdppListFinSet.v [hunk 1]
```diff
@@ -13,7 +13,7 @@ Definition empty_set : set := ∅.
 
 Definition set_add (a : A) (x : set) : set := {[a]} ∪ x.
 
-Definition set_remove (a : A) (x : set) : set := x ∖ {[ a ]}.
+Definition set_remove (a : A) (x : set) : set := x ∖ {[a]}.
```
Comment (palmskog) [inline theories/VLSM/Lib/StdppListFinSet.v:16]: This is harder to read than before.

File: theories/VLSM/Lib/TraceProperties.v [hunk 2]
```diff
@@ -641,32 +641,32 @@ move => [q hq] [p1 hp1] [p2 hp2] h0 tr0 /= h1.
 exact: (@appendT_cont q q p1 p2).
 Qed.
 
-Lemma AppendT_ttA: forall p, p =>> (p *** [|ttA|]).
+Lemma AppendT_ttA : forall p, p =>> (p *** [| ttA |]).
```
Comment (palmskog) [inline theories/VLSM/Lib/TraceProperties.v:644]: The notations here are not the same as elsewhere, they should have no space, please revert the changes for `[|...|]` and `<<..;..>>`.

**Review by palmskog — COMMENTED** [review]
The tactic and definition changes are fine, but this PR also makes changes inside many notations, such as for finsets and traces. Please roll back those changes, which I've pinpointed with comments.

File: theories/VLSM/Lib/ListFinSetExtras.v [hunk 3]
```diff
@@ -12,7 +12,7 @@ Definition empty_set : set := ∅.
 
 Definition set_add (a : A) (x : set) : set := {[a]} ∪ x.
 
-Definition set_remove (a : A) (x : set) : set := x ∖ {[ a ]}.
+Definition set_remove (a : A) (x : set) : set := x ∖ {[a]}.
```
[the remark this answers is not in the document]
Comment (wkolowski) [reply theories/VLSM/Lib/ListFinSetExtras.v:15]: No rationale, it just got hit by the script. I did see it when I was browsing the changes, but it didn't occur to me that it should be reverted.

[the remark this answers is not in the document]
Comment (wkolowski) [reply theories/VLSM/Lib/ListFinSetExtras.v]: @palmskog I updated the script so that it omits this notation.

File: theories/VLSM/Lib/TraceProperties.v (hunk 2 above)
Comment (wkolowski) [reply theories/VLSM/Lib/TraceProperties.v]: Fixed.

**Review by palmskog — APPROVED** [review]
Fine by me.
